src/Core/System/CustomEntity/CustomEntityException.php

```diff
@@ -15,6 +15,7 @@ class CustomEntityException extends HttpException
     public const CUSTOM_FIELDS_AWARE_NO_LABEL_PROPERTY = 'NO_LABEL_PROPERTY';
     public const CUSTOM_FIELDS_AWARE_LABEL_PROPERTY_NOT_DEFINED = 'LABEL_PROPERTY_NOT_DEFINED';
     public const CUSTOM_FIELDS_AWARE_LABEL_PROPERTY_WRONG_TYPE = 'LABEL_PROPERTY_WRONG_TYPE';
+    public const CUSTOM_FIELDS_REFERENCE_TABLE_NOT_FOUND = 'REFERENCE_TABLE_NOT_FOUND';
```

Comment:
```suggestion
    public const REFERENCE_TABLE_NOT_FOUND = 'FRAMEWORK__CUSTOM_ENTITY_REFERENCE_TABLE_NOT_FOUND';
```

Reply:
`CUSTOM_ENTITY_ASSOCIATION_REFERENCE_TABLE_NOT_FOUND` seems to be more clearly, I think.